integration_tests/util.rs

```diff
@@ -435,26 +454,25 @@ pub struct SignetMiner {
 }
 
 impl SignetMiner {
-    pub fn command<CmdArg, Subcommand, SubcommandArg, CmdArgs, SubcommandArgs>(
```

Comment:
Why restrict this to Vec/&str? This is less flexible than before, which handled Vecs/arrays/ &str/String

Reply:
Otherwise, LGTM. I'll merge this, can address this comment in a follow-up PR if necessary